core/kits/views/panel.php

```diff
@@ -13,6 +13,12 @@
 			<# } ); #>
 		</div>
 	</div>
+	<div id="elementor-panel__editor__help">
+		<a id="elementor-panel__editor__help__link" href="https://go.elementor.com/global-settings" target="_blank">
```

Comment:
@shilo-ey  It's the same link for all inner pages?

Reply:
Yes

Reply:
Global Settings - Page that explains Global Settings
Colors/Typo - another page
Theme style - another page and so on.

Every Menu page should have a help doc
@matipojo

Reply:
@udidol what is the status?